verticapy/vdataframe.py

```diff
@@ -6455,13 +6213,16 @@ def first(self, ts: str, offset: str):
             sql_push_ext=self._VERTICAPY_VARIABLES_["sql_push_ext"],
             symbol=self._VERTICAPY_VARIABLES_["symbol"],
         )
-        self.filter("{} <= '{}'".format(ts, first_date))
+        self.filter(f"{ts} <= '{first_date}'")
         return self
 
     # ---#
     @save_verticapy_logs
```

Comment:
shouldnt there be the check_type decorator?

Reply:
good one